music21/musicxml/m21ToXml.py

```diff
@@ -6893,16 +6974,19 @@ def testFullMeasureRest(self):
         self.assertEqual(rest.get('measure'), 'yes')
 
     def testArticulationSpecialCases(self):
-        n = note.Note()
+        n1 = note.Note()
         a = articulations.StringIndication()
-        n.articulations.append(a)
+        n1.articulations.append(a)
+        h = articulations.HammerOn()
+        n1.articulations.append(h)
```

Comment:
This will need a blaring warning/docstring saying *don't do this*, but if you did...